spec/facility_spec.rb

```diff
@@ -23,4 +34,105 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+# VEHICLE REGISTRATION
+  describe 'regular vehicle registration' do
+    it 'can register a regular vehicle' do
```

Comment:
So this is testing that it CAN register a vehicle, but we also want to test the conditions when it can't register a vehicle. So it would be good to have another test where the service is not added to the facility and you expect a different return value.